docs/en/api-guides/usb-serial-jtag-console.rst

```diff
@@ -53,26 +53,26 @@ The USB Serial/JTAG Controller is able to put the {IDF_TARGET_NAME} into downloa
 Limitations
 ===========
 
-There are several limitations to the USB console feature. These may or may not be significant, depending on the type of application being developed, and the development workflow.
+There are several limitations to the USB Serial/JTAG console feature. These may or may not be significant, depending on the type of application being developed, and the development workflow.
```

Comment:
I'm not sure if mentioning JTAG here is less confusing. The JTAG part seems to have nothing to do with the CDC console?

Reply:
Technically they are both "CDC", just implemented using two different peripherals. I thought that the title of the document makes it clear which one we are talking about, but maybe someone would land into the middle of the document from a Google search.

Let's keep the changes you made. If i get disagreement about this during the internal review, i will let you know.